webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  devtool: 'cheap-module-eval-source-map',
+  entry: [
+    'webpack-hot-middleware/client',
+    './app'
+  ],
+  output: {
+    path: path.join(__dirname, 'bundle'),
+    filename: 'app.js',
+    publicPath: '/bundle/'
+  },
+  plugins: [
+    new webpack.optimize.OccurenceOrderPlugin(),
```

Comment:
I dont know about this feature too. I think we can test this feature during developing.